packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -238,10 +282,10 @@ const callbackIfFileIncluded = (meta: Metadata, next: Metadata) => {
  */
 const extractConditionalExpression = (node: t.ConditionalExpression, meta: Metadata): CSSOutput => {
   const conditionalPaths: ['consequent', 'alternate'] = ['consequent', 'alternate'];
-  const css = [];
-  const variables = [];
+  const css: CSSOutput['css'] = [];
+  const variables: CSSOutput['variables'] = [];
 
-  for (const path of conditionalPaths) {
+  const [consequentCss, alternateCss] = conditionalPaths.map((path) => {
     const pathNode = node[path];
     let cssOutput;
 
```

Comment:
Do we still need all the checks before `t.isIdentifier` or can we dumb it down to always try and build the css if it's not an identifier or conditional?

Reply:
Sadly simplifying it here doesn’t play nicely with the logic & assumptions in `buildCss` - it expects things/expressions that we know to contain whole CSS declarations. However, Compiled supports using Conditional Expressions that either return a whole declaration, or just the value side of a declaration.

So we have to do the filtering here to make sure we only try to `buildCss` with whole declarations or more.